assets/stylesheets/back_to_top.css

```diff
@@ -0,0 +1,8 @@
+.back-to-top {
+    bottom: 20px;
+    cursor: pointer;
+    display: none;
+    position: fixed;
+    right: 20px;
+    z-index: 1000;
```

Comment:
Maybe add

```
    border-radius: 50rem;
    width: 2.2rem;
    height: 2.2rem;
```

for consistency with the icon-only buttons we're using in the comment form.

Reply:
For anyone wondering, that makes it a circle. I'll wait for more comments/votes on that.